runtime/lua/vim/fs.lua

```diff
@@ -65,39 +65,61 @@ function M.dir(path)
     vim.loop.fs_scandir(M.normalize(path))
 end
 
---- Find files or directories in the given path.
----
---- Finds any files or directories given in {names} starting from {path}. If
---- {upward} is "true" then the search traverses upward through parent
---- directories; otherwise, the search traverses downward. Note that downward
---- searches are recursive and may search through many directories! If {stop}
---- is non-nil, then the search stops when the directory given in {stop} is
---- reached. The search terminates when {limit} (default 1) matches are found.
---- The search can be narrowed to find only files or or only directories by
---- specifying {type} to be "file" or "directory", respectively.
----
----@param names (string|table|fun(name: string): boolean) Names of the files
----             and directories to find.
----             Must be base names, paths and globs are not supported.
----             If a function it is called per file and dir within the
----             traversed directories to test if they match.
----@param opts (table) Optional keyword arguments:
----                       - path (string): Path to begin searching from. If
----                              omitted, the current working directory is used.
----                       - upward (boolean, default false): If true, search
----                                upward through parent directories. Otherwise,
----                                search through child directories
----                                (recursively).
----                       - stop (string): Stop searching when this directory is
----                              reached. The directory itself is not searched.
----                       - type (string): Find only files ("file") or
----                              directories ("directory"). If omitted, both
----                              files and directories that match {name} are
----                              included.
----                       - limit (number, default 1): Stop the search after
----                               finding this many matches. Use `math.huge` to
----                               place no limit on the number of matches.
----@return (table) The paths of all matching files or directories
+--[[
+
+    Find files or directories in the given path.
+
+    Finds any files or directories given in {names} starting from {path} (if given). If
+    {upward} is "true" then the search traverses upward through parent
+    directories; otherwise, the search traverses downward. Note that downward
+    searches are recursive and may search through many directories! If {stop}
+    is given, then the search stops when the directory given in {stop} is
+    reached. The search terminates when {limit} (1 if omitted) matches are found.
+    The search can be narrowed to find only files or only directories by
+    specifying {type} to be either 'file' or 'directory', respectively.
```

Comment:
Use double quotes instead of single quotes
```suggestion
    specifying {type} to be either "file" or "directory", respectively.
```